mpas_analysis/ocean/climatology_map_bgc.py

```diff
@@ -0,0 +1,356 @@
+# Copyright (c) 2017,  Los Alamos National Security, LLC (LANS)
+# and the University Corporation for Atmospheric Research (UCAR).
+#
+# Unless noted otherwise source code is licensed under the BSD license.
+# Additional copyright and license information can be found in the LICENSE file
+# distributed with this code, or at http://mpas-dev.github.com/license.html
+#
+from __future__ import absolute_import, division, print_function, \
+    unicode_literals
+
+import xarray as xr
+
+from mpas_analysis.shared import AnalysisTask
+
+from mpas_analysis.shared.io.utility import build_config_full_path
+
+from mpas_analysis.shared.climatology import RemapMpasClimatologySubtask, \
+    RemapObservedClimatologySubtask
+
+from mpas_analysis.ocean.plot_climatology_map_subtask import \
+    PlotClimatologyMapSubtask
+
+from mpas_analysis.shared.grid import LatLonGridDescriptor
+
+
+class ClimatologyMapBGC(AnalysisTask):  # {{{
+    """
+    An analysis task for plotting of BGC variables
+
+    Authors
+    -------
+    Phillip J. Wolfram, Riley X. Brady
+
+    """
+    def __init__(self, config, mpasClimatologyTask, afieldName, ampasFieldName,
+                 refConfig=None):  # {{{
+        """
+        Construct the analysis task.
+
+        Parameters
+        ----------
+        config :  ``MpasAnalysisConfigParser``
+            Configuration options
+
+        mpasClimatologyTask : ``MpasClimatologyTask``
+            The task that produced the climatology to be remapped and plotted
+
+        refConfig :  ``MpasAnalysisConfigParser``, optional
+            Configuration options for a reference run (if any)
+
+        Authors
+        -------
+        Phillip J. Wolfram, Riley X. Brady
+        """
+        # call the constructor from the base class (AnalysisTask)
+        super(ClimatologyMapBGC, self).__init__(
+            config=config, taskName='climatologyMapBGC_' + afieldName,
+            componentName='ocean',
+            tags=['climatology', 'horizontalMap', 'BGC', afieldName])
+
+        sectionName = 'climatologyMapBGC'
+        
+        # Clarify that the user is doing preindustrial vs. modern
+        preindustrial = config.getboolean(sectionName, 'preindustrial')
+        if preindustrial:
```

Comment:
@bradyrx, could you create a `setup_and_check` function below and move this bit (including the print statements below) into that function?  That way, this check will only happen and the message will only be printed if the user has requested this specific analysis.  I'm seeing this message even when I do, for example, `./run_mpas_analysis --list`.  In general, the only print statements in the `__init__` function of a task should be related to catastrophic errors.